### DISCUSSION TITLE
Slated for Deprecation?

### DISCUSSION THREAD

**Discussion by BurdetteLamar** [sparklemotion/nokogiri#3371] [General Q&A]
@flavorjones In writing about parsing:  may I say that `XML::RelaxNG.from_document` and `XML::RelaxNG.read_memory` "may be deprecated" (just as similar pairs elsewhere are marked for deprecation)?

(On second thought, `from_document` is not *that* similar to others.)

**Comment by BurdetteLamar** [comment]
Still looking at this.  NM.

**Comment by flavorjones** [comment]
Thank you for asking! Those comments are reminders to contributors in the future. No need to mention it in the documentation right now.
